src/components/RehydrationProvider.js

```diff
@@ -20,7 +21,9 @@ import { App, PageLanding, PageDefinitions, PageInspect, PageHarvest } from './'
 import { omit } from 'lodash'
 import PageAbout from './PageAbout'
 import PageContribution from './PageContribution'
+import PageFullDetail from './PageFullDetail'
 import withTracker from '../utils/withTracker'
+import { FullDetailPage } from './FullDetailView/FullDetailPage'
```

Comment:
The names are a bit confusing `PageFullDetail` and `FullDetailPage`.

Looking at the `<Route...>` all page components start with the word `Page`. So I would suggest renaming `FullDetailPage` -> `PageFullDetail`.
And rename `PageFullDetail` -> `FullDetailContainer`

(double check the names as might have gotten confused)